tests/components/influxdb/test_init.py

```diff
@@ -397,16 +388,16 @@ async def test_event_listener(
         state = MagicMock(
```

Comment:
Maybe we don't need to use the MagicMock anymore? Just set the data directly with `hass.states.async_set`?

Reply:
dropped in 
commit 3e3723129950bae5d387149dc778023cff0ae0a4 (HEAD -> slots_event_bus, upstream/slots_event_bus)
Author: J. Nick Koston <nick@koston.org>
Date:   Wed Jul 5 09:45:56 2023 -0500

    preen7

commit 23f93207f57cd8b941670d57fd3475f09f4e3b1f
Author: J. Nick Koston <nick@koston.org>
Date:   Wed Jul 5 09:44:25 2023 -0500

    preen6

commit e4725805c95e28862305da2ab019dae8338992ff
Author: J. Nick Koston <nick@koston.org>
Date:   Wed Jul 5 09:43:28 2023 -0500

    preen5

commit 07283997de74ea8859837eec28cfe01dde0a1e6d
Author: J. Nick Koston <nick@koston.org>
Date:   Wed Jul 5 09:41:14 2023 -0500

    preen3

commit d9b637c14be82ddc0ca912a7a87044a58dba25c5
Author: J. Nick Koston <nick@koston.org>
Date:   Wed Jul 5 09:40:44 2023 -0500

    preen2

commit 438868c339f70fdb9d709dfaae2771c44bf90f8f
Author: J. Nick Koston <nick@koston.org>
Date:   Wed Jul 5 09:40:04 2023 -0500

    preen1